sdk/typescript/runtime/main.go

```diff
@@ -400,26 +406,34 @@ func (t *TypescriptSdk) detectRuntime(ctx context.Context) (SupportedTSRuntime,
 
 			switch runtime := SupportedTSRuntime(runtime); runtime {
 			case Bun, Node:
-				return runtime, version, nil
+				t.moduleConfig.runtime = runtime
+				t.moduleConfig.runtimeVersion = version
+
+				return nil
+			// Custom base image different from node or bun
 			default:
-				return "", "", fmt.Errorf("detected unknown runtime: %s", runtime)
+				t.moduleConfig.runtime = Node
```

Comment:
It supported for `node` & `bun`

Reply:
No, because only Node has this:

```go
if image != "" {
	ctr = ctr.From(image)
}
```

Bun doesn't:

https://github.com/TomChv/dagger/blob/e56c3629bd9e716f77e1981f86d336e54f59064c/sdk/typescript/runtime/main.go#L253-L258